includes/admin/views/kco-support.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Klarna support tab in the settings page.
+ *
+ * @package  Klarna_Checkout/Includes/Admin/Views
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( isset( $_POST['submit'] ) ) {
+
+	$send = true;
+	if ( ! is_email( $_POST['email'] ) ) {
+		$send = false;
+	}
+
+	$from    = sanitize_email( $_POST['email'] );
+	$to      = 'support@krokedil.se';
+	$subject = sanitize_text_field( $_POST['subject'] );
+	$message = sanitize_textarea_field( $_POST['description'] );
+	$headers = array(
+		'From: ' . $from,
+		'Reply-To: ' . $from,
+		'Content-Type: text/html; charset=UTF-8',
+	);
+
+
+	$attachment = array(
+		WC_LOG_DIR . sanitize_text_field( $_POST['klarna-checkout'] ),
+		WC_LOG_DIR . sanitize_text_field( $_POST['klarna-order-management'] ),
+		WC_LOG_DIR . sanitize_text_field( $_POST['fatal-error-log'] ),
+		WC_LOG_DIR . sanitize_text_field( $_POST['additional-log'] ),
+	);
+
+	foreach ( $_POST as $key => $value ) {
+		if ( strpos( $key, 'additional-log-' ) !== false ) {
+			$attachment[] = WC_LOG_DIR . sanitize_text_field( $value );
+		}
+	}
+
+	$attachment = array_filter(
+		$attachment,
+		function( $path ) {
+			return file_exists( $path ) && is_readable( $path );
+		},
+	);
+
+	if ( 'on' === $_POST['system-report-include'] ) {
+		$system_report = sanitize_textarea_field( $_POST['system-report'] );
+		$filename      = 'system-report-' . date( 'Y-m-d' ) . '.txt';
+		$filepath      = WP_CONTENT_DIR . '/uploads/' . $filename;
+
+		if ( file_put_contents( $filepath, $system_report ) ) {
+			$attachment[] = $filepath;
+		}
+	}
+
+	if ( $send ) {
+		// wp_mail( $to, $subject, $message, $headers, $attachment );
+	}
+}
```

Comment:
You should move logic like this to a separate file instead of having it in the HTML file.

Also you are using the $_POST variable directly, which should never be done without sanitizing it.